workflow-templates/crowdin-download.yml

```diff
@@ -0,0 +1,55 @@
+name: Crowdin download
```

Comment:
From:
https://transferwise.atlassian.net/wiki/spaces/TRAN/pages/1888527079/Moving+to+Crowdin+Enterprise

cc @stephan765